service/src/loader.rs

```diff
@@ -1,106 +1,18 @@
-use std::any::Any;
-use std::ffi::OsStr;
-use std::net::SocketAddr;
-use std::panic::AssertUnwindSafe;
 use std::path::{Path, PathBuf};
```

Comment:
What remains in this file? Should we possibly rename it and rename the feature?

Reply:
Indeed. We still need to build the crate, we still need `check_version` annd `check_no_panic`. I removed `ensure_cdylib`, but we should probably have a `ensure_not_cdylib` function now. Perhaps we can rename it to `builder.rs`/`build.rs`? :thinking:

Reply:
`build.rs` (or something along that) sounds good :+1: